be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -21,7 +22,7 @@ void AggregateBlockingSinkOperator::finish(RuntimeState* state) {
     }
     _is_finished = true;
 
-    if (!_aggregator->is_none_group_by_exprs()) {
+    if (!_aggregator->group_by_expr_ctxs().empty()) {
```

Comment:
use is_none_group_by_exprs instead of group_by_expr_ctxs().empty()

Reply:
fixed